### DISCUSSION TITLE
Doris 1.1 版本 set return_object_data_as_binary = true 后仍然拿不到二进制数据

### DISCUSSION THREAD

**Discussion by mansonliwh** [apache/doris#11484] [A - General / Q&A]
Doris 1.1 版本 set return_object_data_as_binary = true 后，仍然拿不到二进制数据。
在Doris 1.0 版本是可以的。

**Comment by LemonLiTree** [comment]
https://doris.apache.org/zh-CN/docs/faq/sql-faq?_highlight=return_object_data_as_binary#q5-%E6%9F%A5%E8%AF%A2-bitmaphll-%E7%B1%BB%E5%9E%8B%E7%9A%84%E6%95%B0%E6%8D%AE%E8%BF%94%E5%9B%9E-null-%E7%9A%84%E9%97%AE%E9%A2%98
可以看看这个？
